packages/contracts/contracts/core/permission/IPermissionOracle.sol

```diff
@@ -12,10 +12,10 @@ interface IPermissionOracle {
     /// @param _permissionId The permission identifier.
     /// @param _data Optional data passed to the `PermissionOracle` implementation.
     /// @return allowed Returns true if the call is permitted.
-    function hasPermissions(
+    function hasPermission(
```

Comment:
Should we rename this to `isPermitted(...)` as described but it's notice ?

Reply:
To me it makes much less of a difference (hasPermission == isPermitted). But if we have a complete list of all the scopes, maybe it's easier to see?
- DAO => hasPermission(), canUse(), canPerform()
- PermissionManager => isGranted()
- PermissionOracle => isPermitted(), isAuthorized(), ...? Even isGranted()?

If the oracle is conceptually an extension of the PermissionManager, maybe it makes sense to use `isGranted()` in both?

Reply:
amended